multiaccounts/settings/columns_test.go

```diff
@@ -52,7 +52,8 @@ func TestJSONEncoding(t *testing.T) {
 	encoded, err := json.Marshal(settings)
 	require.NoError(t, err)
 
-	require.True(t, strings.Contains(string(encoded), "\"compressed-key\":\"zQ3shudJrBctPznsRLvbsCtvZFTdi3b34uzYDuqE9Wq9m9T1C\""))
+	require.True(t, strings.Contains(string(encoded), "\"compressedKey\":\"zQ3shudJrBctPznsRLvbsCtvZFTdi3b34uzYDuqE9Wq9m9T1C\""))
```

Comment:
If a setting has a column in the settings database table it will have a kebab-case `reactFieldName` associated with it, that's the convention for `Settings`. This is a strange case because `compressed-key` isn't a field of the `Settings` struct or a column of the settings db table.

However we do want to purge all the kebab-case from `status-go` at some point.

See this discussion https://github.com/status-im/status-go/pull/3117#discussion_r1083276062

Reply:
Thanks for sharing. Didn't know about that translation mechanism. So eventually we want to get rid of `reactFieldName`, all kebab cases and just stick to camelCase?

Reply:
Yes, ideally. Convert everything to camelCase.